include/config_defaults.h

```diff
@@ -217,15 +217,15 @@
 #ifndef TOOL_CALCULATOR
 #define TOOL_CALCULATOR 1
 #endif
-#ifndef TOOL_STOPWATCH
-#define TOOL_STOPWATCH 1
-#endif
```

Comment:
Why was this removed? Does your app replace the timer? No. Did you replace the flag? No.

Also you missed one in the `CMakeLists.txt` for the emulator.

Maybe you could introduce a new flag to make the whole "clock-apps" section optional.